From a68446d053d8561085ec7e1f4af69b79fa647b2d Mon Sep 17 00:00:00 2001
From: Ewoud Kohl van Wijngaarden <ekohlvanwijngaarden@antagonist.nl>
Date: Mon, 5 Dec 2016 18:53:36 +0100
Subject: [PATCH] Remove scripts from setup.py

There is already a console_scripts entrypoint which results in a script.
When installing to the system this results in /usr/bin/gitup.py and
/usr/bin/git-up.

Previously it was needed but 222cc6a9910f4fc44fd15a64da5db52a94d9a3c3
removed this requirement by using the full path.
---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index 61a45ed..9aa0b6b 100644
--- a/setup.py
+++ b/setup.py
@@ -5,7 +5,6 @@ setup(
     name="git-up",
     version="1.4.4",
     packages=find_packages(),
-    scripts=['PyGitUp/gitup.py'],
     install_requires=['GitPython>2.0.0', 'colorama>=0.3.7',
                       'termcolor>=1.1.0', 'click>=6.0.0',
                       'six>=1.10.0'],
-- 
2.7.3

